user-guide/Advanced_Modules/User Defined APIs/UD_APIs_Triggering_an_API.md

```diff
@@ -80,6 +80,10 @@ The endpoint itself can also return [errors](#errors) with corresponding status
 
 The API will always return responses encoded in UTF-8.
 
+### CORS
+
+User-defined APIs will not return any CORS headers. It is not safe to trigger user-defined APIs from a web client using e.g. AJAX calls in JavaScript, because there is no way to safely save and use the API tokens there.
```

Comment:
Add a short sentence explaining the consequences. Eg. '....not return any CORS headers. It will thus not be possible to trigger a user-defined API directly from a web client.